### PR TITLE
Small Legion Balance Patch

### PR DESCRIPTION
**Pull request by PlayerAdelheid**
### Ranger-Hunter Centurion Rebalance
Since Legion is meant to be high speed, and since Centurion has a heavy hitter loadout already (M1919), I added a sawn-off AMR that is lighter, can't use a scope, won't knock you down, but it deals less damage and doesn't have 100% AP. It's also kinda inaccurate.

### Venator Sniper Rifle
I gave it extra damage and penetration to make it perform more or less like the veteran ranger's medicine stick, since they are both for the same thing. Previously it was a little too weak, and every venator ditched it for whatever else. Hopefully this will help.

### Mosin Assultus
I added an automatic stripper clip fed mosin nagant for explorers who like to chase their targets instead of staying in the backline, as an alternate rushing loadout. It might need some tweaking after live tests. We'll see.
Due to the nature of the gun, the explorer will only get it and an ammo box. No utility.

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -356,6 +356,28 @@
 		H.visible_message("<span class='danger'>[H] drops to the floor from recoil as they fire unprepared!</span>")
 		user.Knockdown(60)
 
+/obj/item/gun/ballistic/rifle/mag/amrsawn
+	name = "\improper Sawn-off Hécate II AMR"
+	desc = "A Hécate II that has been defiled by someone with too much time on their hands, featuring an extremely short barrel, wooden grip, and restored iron sights. A Legion flag is wrapped around its stock."
+	icon_state = "WHAT"
+	item_state = "amr"
+	mag_type = /obj/item/ammo_box/magazine/amr
+	fire_delay = 12 //Heavy round, tiny bit slower
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/rifle.dm:365]:
```suggestion
	fire_delay = 8
```

**Comment by JasmineRickards** [comment]
I don't think this is a good idea.

**Comment by PlayerAdelheid** [comment]
> I don't think this is a good idea.

Anything specific to tweak?

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 2]
```diff
@@ -1037,6 +1037,32 @@
...
+	icon_prefix = "308"
+	force = 20
+	slowdown = 0.8
+	mag_type = /obj/item/ammo_box/magazine/internal/avtomat
+	fire_delay = 1
+	burst_size = 3
+	spread = 1.2
+	is_automatic = TRUE
+	automatic = 1
+	autofire_shot_delay = 2
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1055]:
```suggestion
	autofire_shot_delay = 2.5
```

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 3]
```diff
@@ -1037,6 +1037,32 @@
...
+	desc = "A pre-war Mosin-Nagant 91/30 rifle was modified into a makeshift machine gun, with a high rate of fire, gas-driven receiver, and increased internal magazine. A Legion bull has been branded into its wooden stock."
+	icon_state = "avtomat"
+	item_state = "rifle"
+	icon_prefix = "308"
+	force = 20
+	slowdown = 0.8
+	mag_type = /obj/item/ammo_box/magazine/internal/avtomat
+	fire_delay = 1
+	burst_size = 3
+	spread = 1.2
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1052]:
```suggestion
	spread = 3
```

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 4]
```diff
@@ -1092,6 +1118,8 @@
 	slowdown = 0.12
 	zoom_amt = 15
 	zoom_out_amt = 17
+	extra_damage = 15 //~39 to 50 damage
+	extra_penetration = 0.4 //On par with medicine stick
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1122]:
```suggestion
	extra_penetration = 0.3 //Slightly less than medicine stick
```

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 5]
```diff
@@ -356,6 +356,28 @@
...
+	name = "\improper Sawn-off Hécate II AMR"
+	desc = "A Hécate II that has been defiled by someone with too much time on their hands, featuring an extremely short barrel, wooden grip, and restored iron sights. A Legion flag is wrapped around its stock."
+	icon_state = "WHAT"
+	item_state = "amr"
+	mag_type = /obj/item/ammo_box/magazine/amr
+	fire_delay = 8
+	recoil = 5
+	spread = 4
+	force = 5
+	slowdown = 0.8
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/rifle.dm:369]:
```suggestion
	slowdown = 0.5
```

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 6]
```diff
@@ -356,6 +356,28 @@
...
+	zoomable = FALSE
+	fire_sound = 'sound/f13weapons/antimaterielfire.ogg'
+	pump_sound = 'sound/f13weapons/antimaterielreload.ogg'
+	extra_damage = -6
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/guns/ballistic/rifle.dm:373]:
```suggestion
	extra_damage = -10
```

**Comment by PlayerAdelheid** [comment]
> I don't think this is a good idea.

How does it look now? I realized the mosin could intrude on some other niches so I made it worse too.

**Pull request merged**